lib/index.js

```diff
@@ -99,7 +99,15 @@ function nodeNetCatSrc(port, input) {
 }
 
 const FUNCTION_PRIORITY = [nativeNC, nodeNC];
-const FLAGS = isElectron() ? ['--ms-enable-electron-run-as-node'] : [];
+const needElectronFlag = vsCodeVersion => {
```

Comment:
You need to update the version right?

Reply:
No, it is from `vscode` api, the vscode runtime, no change on package.json.

Reply:
I mean, you're changing the code right? So this needs to have a new version.

Reply:
We didn't bump the version for sync-rpc previously, and the actual usage of the repo is with commit hash. Do you think we need to bump it?